internal/service/advancedcluster/model_advanced_cluster.go

```diff
@@ -248,6 +252,11 @@ func SchemaAdvancedConfig() *schema.Schema {
 					Optional: true,
 					Computed: true,
 				},
+				"change_stream_options_pre_and_post_images_expire_after_seconds": {
+					Type:     schema.TypeInt,
+					Optional: true,
+					Default:  -1,
```

Comment:
Just to confirm, was the final agreement with the API team that we send the default of -1 if attribute is not defined?

Reply:
-1 and null(not set) is equivalent in this case. Setting the default value to -1 in the schema is necessary for the case of the user wanting to go back to the default value(which might not always be the same) after having set a different value.